### PR TITLE
Fix Python 3 errors when doing ./wpt make-hosts-file

### PR DESCRIPTION
**Pull request by saschanaz**
This does not fix everything, as I was fixing errors by repeating the command and then blocked by pywebsocket. ~~Should we fork it?~~

Edit: Ah, according to https://github.com/web-platform-tests/wpt/issues/9599 the pywebsocket clone is already forked. Maybe it can be modified then :+1: 
Edit 2: Opened #11518.

### CODE REVIEW

**Comment by saschanaz** [comment]
Ah, according to https://github.com/web-platform-tests/wpt/issues/9599 the pywebsocket clone is already forked. Maybe it can be modified then :+1:

**Comment by gsnedders** [comment]
@Ms2ger had also just done a fair bit on getting wptserve running under Py3, though I don't know what the status of that is.

File: tools/wptserve/wptserve/sslutils/__init__.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # flake8: noqa (not ideal, but nicer than adding noqa: F401 to every line!)
```
Comment (Ms2ger) [inline tools/wptserve/wptserve/sslutils/__init__.py]: @gsnedders you added this; do you remember why you needed to keep the openssl/pregenerated imports?

Either we should remove the noqa annotation, or we should keep the imports. (I'd merge immediately if you did the latter; need more info to approve the former.)

**Review by Ms2ger — CHANGES_REQUESTED** [review]
LGTM besides one issue.

**Comment by Ms2ger** [comment]
My code is in https://github.com/Ms2ger/web-platform-tests/tree/iteritems-testfiles and https://github.com/Ms2ger/web-platform-tests/tree/py3-resources-test ; feel free to pick up changes from there.

File: tools/wptserve/wptserve/sslutils/__init__.py (hunk 1 above)
Comment (gsnedders) [reply tools/wptserve/wptserve/sslutils/__init__.py]: I have no idea. I was trying to change as little as possible.

**Comment by gsnedders** [comment]
How practical would it be to remove the `wptserve`/`serve` exceptions from `tools/conftest.py` to make sure we don't regress this? Even if we have to mark almost every test as xfail on Py3!

File: tools/wptserve/wptserve/sslutils/__init__.py (hunk 1 above)
Comment (saschanaz) [reply tools/wptserve/wptserve/sslutils/__init__.py]: >was trying to change as little as possible.

It seems okay to merge then 😁

Comment (Ms2ger) [reply tools/wptserve/wptserve/sslutils/__init__.py]: I offered two options in my comment; please implement one of them.

Comment (saschanaz) [reply tools/wptserve/wptserve/sslutils/__init__.py]: Done!

**Review by Ms2ger — APPROVED** [review]

**Comment by gsnedders** [comment]
Interestingly `test_make_hosts_file_*` still fail despite the command working!

**Comment by gsnedders** [comment]
Uh, no, that's wrong. I… ran that using Py2.

**Comment by gsnedders** [comment]
#11669 makes us run tests on Py3.

**Pull request merged**